### PR TITLE
Remote control loadgen

### LINKED ISSUE #14: add remote-control feature
For the next testnet phase, we plan to recruit all validator operators to run load-generator nodes. They'll be asked to point the loadgen at their own (local) RPC server, which will hopefully fix some of the problems we've had with RPC servers falling over.

To manage this, we want to add a central coordinator. Each generator will be run with a coordinator URL and a nickname (their validator moniker) as `argv` arguments. The generator will establish a WebSocket connection to this URL and register itself for updates (instead of / in addition to the local URL listener). We'll run the server, and when we want to crank up the load, the server will tell some subset of validators to run faster.

We should include status updates on this websocket channel: each time a load operation completes, so we can measure the overall offered/accepted load.

The coordination server can live in this repository. The control port should have a simple API token to protect access. We won't bother applying any protection to the client connections.

### PR DESCRIPTION
**Pull request by mhofman**
This PR adds the first bits to remote control loadgen instances through firebase.
Closes #14 

TODO:
- [X] Report task executions
- [ ] Security rules to avoid unexpected changes
- [ ] Orchestrator to update the requested loadgen configs

### CODE REVIEW

File: loadgen/package.json [hunk 1]
```diff
@@ -30,10 +30,12 @@
     "@agoric/install-ses": "*",
     "@agoric/marshal": "*",
     "@agoric/notifier": "*",
+    "@agoric/promise-kit": "*",
     "@agoric/store": "*",
     "@agoric/ui-components": "*",
     "@agoric/zoe": "*",
-    "esm": "^3.2.5"
+    "esm": "^3.2.5",
+    "firebase": "^9.0.0-beta.8"
```
Comment (warner) [inline loadgen/package.json:38]: we need an unreleased version of firebase? that's.. exciting

**Review by warner — APPROVED** [review]
I only looked at it briefly, but it seems pretty good.

File: loadgen/package.json (hunk 1 above)
Comment (mhofman) [reply loadgen/package.json:38]: Yeah, v9 is modular and avoid loading everything, which I figured would be less problematic in a SES environment.

**Comment by mhofman** [comment]
@warner, I believe this is ready for merge so that validators can pull the latest. The backend changes can be done in a follow up PR.

**Pull request merged**